### ISSUE TITLE
Cannot specify the background by config in vega.parse

### ISSUE THREAD

**Issue by yhoonkim** [vega/vega#1040]
I tried to change the background via `config` in `vega.parse` but failed. 

**Config**
```json
{
    "background": "red",
    "group": {
      "fill": "#dedede"
    }
}
```
Here are the returns: 
- Without `config`
![image](https://user-images.githubusercontent.com/6269456/31694167-8c04897e-b357-11e7-84bc-0c871da450bb.png)
- Canvas with `config`
![image](https://user-images.githubusercontent.com/6269456/31694195-c03a695c-b357-11e7-8baa-bacc5ec725a7.png)
- SVG with `config`
![image](https://user-images.githubusercontent.com/6269456/31694209-d240a788-b357-11e7-8849-ea94c891391b.png)

**Vega spec**
```json
{
    "$schema": "https://vega.github.io/schema/vega/v3.0.json",
    "width": 200,
    "height": 200,
    "padding": 5,

    "data": [
      {
        "name": "table",
        "values": [
          {"category": "A", "amount": 28},
          {"category": "B", "amount": 14}
        ]
      }
    ],

    "scales": [
      {
        "name": "xscale",
        "type": "band",
        "domain": {"data": "table", "field": "category"},
        "range": "width",
        "padding": 0.05,
        "round": true
      },
      {
        "name": "yscale",
        "domain": {"data": "table", "field": "amount"},
        "nice": true,
        "range": "height"
      }
    ],

    "axes": [
      { "orient": "bottom", "scale": "xscale" },
      { "orient": "left", "scale": "yscale" }
    ],

    "marks": [
      {
        "type": "rect",
        "from": {"data":"table"},
        "encode": {
          "enter": {
            "x": {"scale": "xscale", "field": "category"},
            "width": {"scale": "xscale", "band": 1},
            "y": {"scale": "yscale", "field": "amount"},
            "y2": {"scale": "yscale", "value": 0}
          },
          "update": {
            "fill": {"value": "steelblue"}
          },
          "hover": {
            "fill": {"value": "red"}
          }
        }
      }
    ]
  }
```

**Comment by domoritz** [comment]
I remember that this worked in the past but now the background on canvas is just black.

**Comment by jheer** [comment]
Fixed by changes in [vega-parser v2.3.0](https://github.com/vega/vega-parser/releases/tag/v2.3.0). Will be included in the next Vega release.
